crates/core/tedge_agent/src/agent.rs

```diff
@@ -205,12 +210,11 @@ impl SmAgent {
     async fn subscribe_and_process(
```

Comment:
Since this function doesn't do `subscribe` anymore, calling this function `subscribe_and_process` looks misleading.